public/config/configuration.json

```diff
@@ -3,6 +3,7 @@
   "AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca",
   "AUTH_API_VERSION": "/api/v1",
   "AUTH_WEB_URL": "https://dev.bcregistry.ca/business/auth/",
+  "BCROS_HOME_URL": "https://yfthig-dev.web.app/",
```

Comment:
In the release ticket for these changes, please call out the environment changes for Dev, Test and Prod!

Reply:
OK, I know you're on this already 👍